File: app/src/main/java/com/evaluation/model/result/dob/Dob.java
package com.evaluation.model.result.dob;

import com.google.gson.annotations.SerializedName;

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
public class Dob {

    @SerializedName("date")
    private String mDate;

    @SerializedName("age")
    private Integer mAge;

    public String getDate() {
        return mDate;
    }

    public Integer getAge() {
        return mAge;
    }
}

File: app/src/main/java/com/evaluation/model/result/dob/Dob.kt
package com.evaluation.model.result.dob

import com.google.gson.annotations.SerializedName

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
data class Dob(
    @SerializedName("date")
    val mDate: String,

    @SerializedName("age")
    val mAge: Int
)